example/deployment/AWS-EC2-S3/README.md

```diff
@@ -0,0 +1,120 @@
+  aws_session_token = "<Your AWS subscription session token>"
+  aws_profile_name = "<Your AWS profile name>"
+
+  bucket_name = "<Globally unique S3 bucket name>" #Only lowercase alphanumeric characters and hypens allowed
+  key_name = "<Name of EC2 key pair instance for EC2 cluster>"
+  pem_file = "/path/to/created/private/key.pem"
+  ingress_ip_range = ["1.2.3.4/32", "2.3.4.5/32"]
+  instance_name = "PalisadeExample"
+  instance_type = "t2.large"
+  palisade_version = "<The Version of Palisade you want to run>"
+```
+This needs to be populated with the values indicated above before the example will run.
+1. Your AWS session token, access key and secret key are specific to your AWS subscription. You can find information on where these are located [here](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html). 
+    You can setup a named profile according to instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html).
+    
+    You should set **either** the access and secret keys plus session token **or** a profile name, not both. If you are using shared credentials, the profile name should be in the AWS shared credential file
+    `~/.aws/credentials` (or `%UserProfile%\.aws\credentials` on Windows). 
+                
+    2. The `bucket_name` must be a globally unique S3 bucket name of your choosing. Bucket naming rules are [here](https://docs.aws.amazon.com/AmazonS3/latest/dev//BucketRestrictions.html#bucketnamingrules).
+    3. You will need to create a key pair which allows you to SSH into the EMR cluster and for Terraform to provision the cluster:
+        1. Create a key pair by logging in and from the AWS console, select Services -> EC2. Then select Key Pairs from the side menu.
+        2. Click "Create Key Pair" which will ask for a name and give you a private certificate to download.
+        
+       Put the name of your key pair in the `key_name` field  as it appears in AWS (no `.pem` extension).
+    
+    4. `pem_file` should be the path to the private key create in previous step (ending in `.pem` extension).
+    6. The `ingress_ip_range` is a list of public IPs that will be able to connect into the EMR cluster. This should be your public IP for your client machine.
+
+4. Create a symbolic link to the above terraform.tfvars file in example/deployment/AWS-EC2-S3/terraform-scripts/InstanceRunningExample
+```bash
+  cd example/deployment/AWS-EC2-S3/terraform-scripts/InstanceRunningExample; ln -s ../InstanceRunningPalisade/terraform.tfvars terraform.tfvars; cd -
+```
+
+5. Run the Terraform to create the two EC2 instances, install and start up Palisade on the first instance, install the example on the second instance and run it.
+
+    **You will start to incur charges to your AWS subscription at this point!**
+```bash
+  ./example/deployment/AWS-EC2-S3/bash-scripts/runAWS-EC2-S3Example.sh
+```
+
+After completion run terraform destroy to tear down the two EC2 instances:
```

Comment:
Probably best to move this to the end of the README, otherwise someone is bound to run the destroy and then try to run the example.